cvm/cert-manager/nginx_conf/https.conf

```diff
@@ -22,6 +33,20 @@ server {
 
     # Remote attestation service
     location = /tdx_quote {
+        # Handle preflight OPTIONS requests
+        if ($request_method = 'OPTIONS') {
+            add_header 'Access-Control-Allow-Origin' $cors_origin always;
+            add_header 'Access-Control-Allow-Methods' 'GET, POST, OPTIONS' always;
+            add_header 'Access-Control-Max-Age' 1728000 always;
+            add_header 'Content-Type' 'text/plain; charset=utf-8' always;
+            add_header 'Content-Length' 0 always;
+            return 204;
+        }
```

Comment:
I have seen `Access-Control-Allow-Headers` as well being used. We don't need that?

Reply:
My understanding is that those have good default, so no need initially to set them.